aksel.nav.no/website/components/token-docs/toolbar/SearchField.utils.ts

```diff
@@ -0,0 +1,30 @@
+import Fuse from "fuse.js";
+import { tokens } from "@navikt/ds-tokens/token_docs";
+import { TokenForDocumentationT } from "../../types/tokens";
+
+function tokenIndex(_tokens: TokenForDocumentationT[]) {
+  return new Fuse(_tokens, {
+    threshold: 0.1,
+    keys: [
```

Comment:
Weighting is "random". May adjust if needed? I don't think it matters a lot, as search is pretty much hit-or-miss.